### PR TITLE
Drop support for Python 2.7, 3.5, 3.6.

### PR DESCRIPTION
**Pull request by dataflake**

### CODE REVIEW

File: src/Shared/DC/ZRDB/sqlvar.py [hunk 1]
```diff
@@ -48,14 +48,12 @@
...
 
-import six
-
 from DocumentTemplate.DT_Util import ParseError
 from DocumentTemplate.DT_Util import name_param
 from DocumentTemplate.DT_Util import parse_params
 
 
-StringTypes = six.string_types + (six.binary_type,)
+StringTypes = (str, bytes)
```
Comment (icemac) [inline src/Shared/DC/ZRDB/sqlvar.py:56]: The name is a bit questionable as `bytes` are no strings, but I think we have to keep it this way.

File: src/Shared/DC/ZRDB/RDB.py [hunk 2]
```diff
@@ -34,8 +32,7 @@ def parse_text(s):
     return '\\'.join(r)
 
 
-if six.PY3:
-    long = int
+long = int
```
Comment (icemac) [inline src/Shared/DC/ZRDB/RDB.py:35]: I'd appreciate replacing `long` with `int` in this module.

File: CHANGES.rst [hunk 3]
```diff
@@ -1,8 +1,10 @@
 Changelog
 =========
 
-3.17 (unreleased)
------------------
+4.0 (unreleased)
+----------------
+
+- Drop support for Python 2.7, 3.5, 3.6.
```
Comment (icemac) [inline CHANGES.rst:7]: Could you please also mention the removal of the deprecated `Surrogate` class?

**Review by icemac — APPROVED** [review]
Mostly LGTM.

File: CHANGES.rst (hunk 3 above)
Comment (dataflake) [reply CHANGES.rst:7]: Done

File: src/Shared/DC/ZRDB/RDB.py (hunk 2 above)
Comment (dataflake) [reply src/Shared/DC/ZRDB/RDB.py]: Done

**Pull request merged**